rules/windows/defense_evasion_defendercontrol_activity.toml

```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2020/12/23"
+maturity = "production"
+updated_date = "2021/1/26"
+
+[rule]
+author = ["Elastic"]
+description = "DefenderControl is a portable utility used to disable Windows Defender through the use of the command-line or a GUI."
+false_positives = [
+    "DefenderControl is a legitimate tool that could be used by administrators to perform specific functions.",
+]
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "DefenderControl Activity Detected"
```

Comment:
since its switched to a registry rule, we probably don't need to focus specifically on `DefenderControl` for the name of the rule and in the description. Maybe something like `Windows Defender Disabled via Registry Modification`